package.json

```diff
@@ -17,12 +17,17 @@
     "develop-test-runtime": "cd test-site && yarn develop",
     "build-test-runtime": "cd test-site && yarn build",
     "serve-test-runtime": "cd test-site && yarn serve",
-    "test-schema": "yarn clean-test-runtime && START_SERVER=true jest ./test-site/__tests__/integration/schema && WPGQL_INCREMENT=true START_SERVER=true jest ./test-site/__tests__/integration/schema",
-    "test-build": "yarn clean-test-runtime && jest ./test-site/__tests__/integration/build && WPGQL_INCREMENT=true jest ./test-site/__tests__/integration/build",
-    "test-build-default-options": "yarn clean-test-runtime && DEFAULT_PLUGIN_OPTIONS=true jest ./test-site/__tests__/integration/build && WPGQL_INCREMENT=true DEFAULT_PLUGIN_OPTIONS=true jest ./test-site/__tests__/integration/build",
-    "test": "yarn build && yarn test-schema && yarn test-build && yarn test-build-default-options && yarn clean-test-runtime",
-    "test-schema-watch": "SHOW_GATSBY_PROCESS_STDOUT=true jest --watch ./test-site/__tests__/integration/schema",
-    "test-build-watch": "SHOW_GATSBY_PROCESS_STDOUT=true jest --watch ./test-site/__tests__/integration/build",
+    "ci-test": "server-test 'yarn docker-start' http://localhost:8001",
+    "ci": "yarn lint && yarn ci-test",
+    "docker-start": "docker-compose up",
+    "test-schema-first": "START_SERVER=true jest ./test-site/__tests__/integration/schema",
+    "test-schema-increment": "WPGQL_INCREMENT=true START_SERVER=true jest ./test-site/__tests__/integration/schema",
+    "test-schema": "yarn clean-test-runtime && yarn test-schema-first && yarn test-schema-increment",
+    "test-build": "DEFAULT_PLUGIN_OPTIONS=true jest ./test-site/__tests__/integration/build",
+    "test": "yarn build && yarn test-schema && yarn test-build && yarn clean-test-runtime",
+    "test-schema-watch": "SHOW_GATSBY_PROCESS_STDOUT=true yarn test-schema-first --watch",
+    "test-build-watch": "SHOW_GATSBY_PROCESS_STDOUT=true yarn test-build --watch",
+    "test-update": "yarn test-schema-first -u && yarn test-schema-increment -u && yarn test-build -u",
```

Comment:
I think we want to remove the `-u` flags here

Reply:
@TylerBarnes they are there on purpose!

Reply:
Ahhh 😆 oops, I didn't read the script name. That's useful!